wireless_modules/mqtt_test/config.example.py

```diff
@@ -0,0 +1,4 @@
+ESSID = "<WIFI NETWORK NAME HERE>"
```

Comment:
Can these values be read into from a `.env` file instead? If we can, we can have a `config.py` script that just reads the values there.

Reply:
Just looked it up and it doesn't seem like you are able to use .env files without having to read it just like a normal file.